drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -102,15 +102,30 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
 
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
-        ChromeOptions chromeOptions = new ChromeOptions();
-        CapabilitiesOptionsMapper.mapCapabilities(chromeOptions, capabilities, BROWSER_CAPABILITIES);
-
         // verify binary capabilities
         if (Validate.nonEmpty(binary)) {
```

Comment:
`performValidations` to check if the binary is executable is set as a separate condition with a separate if-clause. Doesn't it set the chromeOptions irrespective of whether the binary is executable or not as per the original logic ?